tests/check_check_log_internal.c

```diff
@@ -72,4 +72,3 @@ Suite *make_log_internal_suite(void)
   
   return s;
 }
-
```

Comment:
Currently (without my commit) there are 2 empty lines. The view of GitHub hides the last one. Check the file with `cat`.